src/plugins/github/graphql.js

```diff
@@ -631,6 +631,11 @@ function mergeDirect<T>(destination: T, source: any): T {
   }
 }
 
+// If a user deletes their account, then the author is null
+// (the UI will show that the message was written by @ghost).
+// Therefore, NullableAuthorJSON is preferred to AuthorJSON
+// for most actual usage.
+export type NullableAuthorJSON = AuthorJSON | null;
```

Comment:
An alternate way to do this would be to export only `NullableAuthorJSON`
(possibly even renamed to `AuthorJSON`), and ask clients to write
`$NonMaybeType<NullableAuthorJSON>` when they want the non-null version.
I suspect that this would incur fewer accidental uses of `AuthorJSON`,
but I don’t have a strong preference.